src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -165,7 +165,7 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
                 if (modelBytes == null)
                 {
                     Host.CheckNonWhiteSpace(options.ModelFile, nameof(options.ModelFile));
-                    Host.CheckUserArg(File.Exists(options.ModelFile), nameof(options.ModelFile));
+                    Host.CheckIO(File.Exists(options.ModelFile), "Model file path {0} doesnot exists.", options.ModelFile);
```

Comment:
Missing space: (also the file could be missing, not just the path to it)
```suggestion
                    Host.CheckIO(File.Exists(options.ModelFile), "Model file {0} does not exists.", options.ModelFile);
```